easybuild/easyconfigs/j/juliaup/juliaup-1.14.7-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,37 @@
+# Paul Melis, SURF (paul.melis@surf.nl)
+easyblock = 'Binary'
+
+name = 'juliaup'
+version = '1.14.7'
+
+homepage = 'https://github.com/JuliaLang/juliaup'
+description = """A cross-platform installer for the Julia programming language."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/JuliaLang/juliaup/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['8fda7c7af09543f67279e5ccbb7939ca61e0d1a9846cb366c3a435e03abb8b1b']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('Rust', '1.75.0'),
+]
+
+extract_sources = True
+
+install_cmd = "cargo build --release && cargo install --root %(installdir)s --path ."
```

Comment:
We have specific support for cargo packages in EasyBuild, please use the `Cargo` easyblock in this easyconfig of `juliaup`. You can find an example in [ripunzip-0.4.0.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/r/ripunzip/ripunzip-0.4.0.eb)

You can generate the list of `crates` from the `Cargo.lock` file in juliaup sources with the command
```
python -m easybuild.easyblocks.generic.cargo /path/to/juliaup/
```

Reply:
Okay, updated

Reply:
Hmm, I see checks failing due to missing checksums for the crates. @lexming How can I generate those values easily?

Reply:
The `Cargo` easyblock can do that for you from the `Cargo.toml` file. Use the command: `python -m easybuild.easyblocks.generic.cargo /path/to/Cargo.toml`

Reply:
Ah indeed, sorry I misread your comment. Once you have the list of crates in the easyconfig, you can inject the checksums with `eb --force --inject-checksums=sha256 /path/to/easyconfig.eb`

Reply:
Thanks, just generated the checksums! Let's see if it passes the tests now.